### PR TITLE
Added Negotiation and Scrounge Skill Settings for Administrator Personnel

### PR DESCRIPTION
**Pull request by IllianiBird**
**Description:**
This PR introduces new features to enhance the capabilities of administrator personnel by enabling them to optionally generate with the Negotiation and Scrounge skills. These features include the option to generate administrators with these skills and to factor these skills into their experience level calculations. 

**Changes Include:**
1. **Negotiation Skill for Administrators:**
   - Administrators can now optionally generate with the Negotiation skill.

2. **Scrounging Skill for Administrators:**
   - Administrators can now optionally generate with the Scrounge skill.

3. **Experience Level Calculations:**
   - The experience level calculations for administrators now optionally include the Negotiation and Scrounge skills.
   - This allows for a more comprehensive evaluation of an administrator's capabilities and experience.

**Implementation Details:**
- Updated the administrator personnel generation logic to include Negotiation and Scrounge skills as optional attributes.
- Modified the experience calculation algorithm to optionally account for the new skills.

**Related Issue:**
- Must be reviewed and merged after #4054 as this PR was forked from that branch.

**Testing:**
- Verified that administrators can be generated with Negotiation and Scrounge skills.
- Confirmed that these skills are correctly included in experience level calculations when enabled.
- Ensured no adverse effects on other functionalities or personnel types.

### CODE REVIEW

File: MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java [hunk 1]
```diff
@@ -345,6 +351,24 @@
                 }
                 break;
             }
+            case CMD_BEGIN_EDUCATION: {
+                for (Person person : people) {
+                    EducationController.beginEducation(gui.getCampaign(), person, data[1], data[2], Integer.parseInt(data[3]), data[4], data[5]);
```
[the remark this answers is not in the document]
Comment (IllianiCBT) [reply MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java:356]: false positive from Education Module

**Comment by HammerGS** [comment]
Also picked up conflicts.

**Comment by IllianiCBT** [comment]
Merged `module education` branch into `Admin_addNegotiationAndScrounge` to resolve conflicts.

This resulted from a misunderstanding of how branching from a branch works. I thought it would dynamically replicate changes made in the parent branch to all child branches. Instead, it creates a snapshot of the the parent branch when the child branch is created.

This means that there will be a lot of repeat commits in both this branch, and the eventual `module_turnover` branch (once that is PR ready). This shouldn't cause any problems, but will look ugly. Mea culpa.

**Comment by Sleet01** [comment]
> Merged `module education` branch into `Admin_addNegotiationAndScrounge` to resolve conflicts.
> 
> This resulted from a misunderstanding of how branching from a branch works. I thought it would dynamically replicate changes made in the parent branch to all child branches. Instead, it creates a snapshot of the the parent branch when the child branch is created.
> 
> This means that there will be a lot of repeat commits in both this branch, and the eventual `module_turnover` branch (once that is PR ready). This shouldn't cause any problems, but will look ugly. Mea culpa.

`git rebase` is the tool to use to bring base-branch updates into an offshoot branch seamlessly.  It will automatically merge the sub-branch changes on top of updated base branch code state, or prompt if there is a conflict.  I usually rebase my branches prior to submitting PRs if development takes more than a couple days.

**Review by Sleet01 — APPROVED** [review]
LGTM

**Pull request merged**